oe_theme.theme

```diff
@@ -245,6 +245,18 @@ function oe_theme_preprocess_form_element(array &$variables): void {
  * Add paragraph formatting to body.
  */
 function oe_theme_preprocess_field(&$variables, $hook): void {
+  // Add space between paragraphs.
```

Comment:
Could have more information concerning the class chose?
for example the link of the ECL2 marge documentation.